src/pages/EntityDetails/Model/Secrets/SecretsTable/SecretsTable.tsx

```diff
@@ -68,20 +44,86 @@ const SecretsTable = () => {
     if (!secrets) {
       return [];
     }
-    return secrets.map((secret) => ({
-      name: secret.label,
-      id: secret.uri.replace(/^secret:/, ""),
-      revision: secret.revisions[0]?.revision,
-      description: secret.description,
-      owner: secret["owner-tag"]?.startsWith("model-")
-        ? "Model"
-        : secret["owner-tag"],
-      created: <RelativeDate datetime={secret["create-time"]} />,
-      updated: <RelativeDate datetime={secret["update-time"]} />,
-      actions: "",
-    }));
+    return secrets.map((secret) => {
+      const id = secret.uri.replace(/^secret:/, "");
+      return {
+        name: secret.label,
+        id: (
+          <div className="u-flex u-flex--gap-small">
+            <TruncatedTooltip
+              wrapperClassName="u-flex-shrink u-truncate"
+              message={id}
+            >
+              {id}
+            </TruncatedTooltip>
+            <div className="has-hover__hover-state">
+              <Tooltip message="Copy secret URI">
+                <Button
+                  appearance="base"
+                  className="is-small"
+                  onClick={() => {
+                    copyToClipboard(secret.uri);
```

Comment:
Left this comment here just to double check if we want to copy the entire uri or just the part after "...secret:"?

Reply:
Yep this means they can paste the full URI for CLI commands or when configuring apps.